Libraries/src/Amazon.Lambda.RuntimeSupport/Bootstrap/InvokeDelegateBuilder.cs

```diff
@@ -320,7 +320,7 @@ private Expression CreateSerializeExpression(object customerSerializerInstance,
                 if (customerSerializerInstance == null)
                 {
                     throw LambdaExceptions.ValidationException(Errors.UserCodeLoader.SerializeMissingAttribute,
-                        _handler.AssemblyName, _handler.MethodName, dataType.FullName);
```

Comment:
slight perf, we should cache the `typeof(LambdaSerializerAttribute)` in a member to avoid performing same operation. Not ship blocking.